README.md

```diff
@@ -19,8 +19,10 @@ Playbooks are located in the `playbooks/` directory.
 | Playbook                                 | Inventory                             | Purpose                                                            |
 | ---------------------------------------- | ------------------------------------- | ------------------------------------------------------------------ |
 | `virthost-setup.yml`                     | `./inventory/virthost/`               | Provision a virtual machine host                                   |
-| `kube-install.yml`                       | `./inventory/vms.local.generated`     | Install and configure a k8s cluster                                |
-| `kube-teardown.yml`                      | `./inventory/vms.local.generated`     | Runs `kubeadm reset` on all nodes to tear down k8s                 |
+| `bmhost-setup.yml`                       | `./inventory/bmhost/`                 | Provision a bare metal host and add to group nodes.                |
+| `allost-setup.yml`                       | `./inventory/allhosts/`               | Provision both a virtual machine host and a bare metal host.       |
+| `kube-install.yml`                       | `./inventory/all.local.generated`     | Install and configure a k8s cluster using all hosts in group nodes |
```

Comment:
how about to change as following?
`./inventory/vms.local.generat` for libvirt, `./inventory/all.local.generated` for baremetal

Reply:
Tomo, thanks for your review. bmhot-setup provisions the bare metal host only. The idea is to add the bare-metal node to [nodes] group after the setup. The idea is that once, the hosts are provisioned, kube-install.yml can be run to set up k8s on either bm hosts, vm hosts or a combination of both.

Reply:
Hi, I just mention that 'kube-install.yml' and 'kube-teardown.yml', not 'bmhost-setup.yml'.

Reply:
Check PR #293 to see if it is clearer? Thanks.